### PR TITLE
+manage friends  and request friends

### PR DESCRIPTION
**Pull request by giapnguyenitf**

### CODE REVIEW

**Comment by giapnguyenitf** [comment]
ready

File: app/Http/Controllers/FriendController.php [hunk 1]
```diff
@@ -60,4 +60,34 @@ public function update($id)
...
+            $query->select('second_user')->from('friendships')
+            ->where(function ($query) {
+                $query->where('first_user', Auth::user()->id)->where('status', config('setting.true'));
+            });
+        })->orWhereIn('id', function ($query) {
+            $query->select('first_user')->from('friendships')
+            ->where(function ($query) {
+                $query->where('second_user', Auth::user()->id)->where('status', config('setting.true'));
+            });
+        })->paginate(10);
```
Comment (trangkarol) [inline app/Http/Controllers/FriendController.php]: Đưa `10` vào file config setting

File: app/Http/Controllers/FriendController.php [hunk 2]
```diff
@@ -60,4 +60,34 @@ public function update($id)
...
+    }
+
+    public function showRequests()
+    {
+        $friends_requests = User::whereIn('id', function ($query) {
+            $query->select('first_user')->from('friendships')
+            ->where(function ($query) {
+                $query->where('second_user', Auth::user()->id)->where('status', config('setting.false'));
+            });
+        })->paginate(10);
```
Comment (trangkarol) [inline app/Http/Controllers/FriendController.php]: Đưa `10` vào file config setting

File: resources/views/user/friendsRequests.blade.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                    @endforeach
+                @else
+                    <div class="no-request">
+                        <h5>@lang('label.no_friends_requests')</h5>
+                    </div>
+                @endif
+            </div>
+             <div class="col-md-12">
+                 <div  class="paginate-bar">
+                 {{ $friends_requests->links() }}
```
Comment (trangkarol) [inline resources/views/user/friendsRequests.blade.php]: Indent: 4 space

**Comment by giapnguyenitf** [comment]
ready

**Pull request merged**